assets/model_monitoring/components/src/model_data_collector_preprocessor/genai_preprocessor_df_schemas.py

```diff
@@ -38,8 +38,8 @@ def _get_aggregated_trace_log_spark_df_schema() -> StructType:
             StructField("session_id", StringType(), True),
             StructField("start_time", TimestampType(), False),
             StructField("end_time", TimestampType(), False),
-            StructField("input", StringType(), False),
-            StructField("output", StringType(), False),
+            StructField("input", StringType(), True),
```

Comment:
this is all const values, why not define a const like variable, e.g. PREPROCSSED_SPA_SCHEMA, AGGREGATED_TRACE_SCHEMA?
Under current implementation, the const schema is constructed every time you call this function, which is un-necessary

Reply:
followed the output schema pattern from another component. Can code refactor later